bdd/features/e2e/E2E-014-python.feature

```diff
@@ -0,0 +1,118 @@
+Feature: Test our shiny new Python runner
+
+    @python
+    Scenario: E2E-014 TC-001 Run simple python sequence with input and output
+        Given host is running
+        When sequence "../python-hello.tar.gz" loaded
```

Comment:
To continue the thought: This would then not point to something like

`When the "hello" sequence is loaded`

It could even point to specific tgz file.

Reply:
Doing this properly would mean reorganizing much of the BDD tests, and would take, I'd say, 4 days. I don't think we have that much time, and we don't want to risk upsetting current testing that @alicja-gruzdz and @a-tylenda are doing.